main.py

```diff
@@ -137,8 +137,8 @@ async def on_message(message: discord.Message):
                 # default true
                 pass
 
-            # don't respond to bot's own posts or if message reacts are disabled
-            if message.author == self.client.user or not message_reacts_enabled:
+            # don't respond to bots, bot's own posts or if message reacts are disabled
+            if message.author == self.client.user or not message_reacts_enabled or message.author.bot:

```

Comment:
```suggestion
            if (message.author == self.client.user) or (not message_reacts_enabled) or message.author.bot:

```
To prevent logical ambiguity